rollbar/__init__.py

```diff
@@ -779,76 +780,52 @@ def _add_locals_data(data, exc_info):
             frame_num += 1
             continue
 
-        # Create placeholders for args/kwargs/locals
-        args = []
-        kw = {}
+        # Create placeholders for argspec/varargspec/keywordspec/locals
+        argspec = []
```

Comment:
You can probably just set this to `None` like the others and change the `extend()` below to an assignment.